packages/pothos-graphql/src/WebPageSummary/index.ts

```diff
@@ -60,8 +60,8 @@ builder.queryField('allSummaries', (t) =>
           `),
           {},
         )
-        const webPageSummarydatas = result.webPageSummaries?.data ?? []
-        return webPageSummarydatas.map((data) => {
+        const webPageSummaryDatas = result.webPageSummaries?.data ?? []
```

Comment:
There is no Datas in english language

```suggestion
        const webPageSummariesData = result.webPageSummaries?.data ?? []
```